snapshots/worldbank_wdi/2025-01-24/wdi.zip.dvc

```diff
@@ -0,0 +1,24 @@
+meta:
+  origin:
+    producer: World Bank
+    title: World Development Indicators
+    description: |-
+      The World Development Indicators (WDI) is the primary World Bank collection of development indicators, compiled from officially-recognized international sources. It presents the most current and accurate global development data available, and includes national, regional and global estimates.
+    citation_full: World Bank's World Development Indicators (WDI).
+    attribution: Multiple sources compiled by World Bank (2025)
+    url_main: https://datacatalog.worldbank.org/search/dataset/0037712/World-Development-Indicators
+    url_download: |-
+      https://datacatalogfiles.worldbank.org/ddh-published/0037712/DR0045575/WDI_CSV_2024_10_24.zip?versionId=2024-12-18T05:42:41.1131485Z
```

Comment:
I see the last link is https://datacatalogfiles.worldbank.org/ddh-published/0037712/DR0045575/WDI_CSV_2024_12_16.zip?versionId=2025-01-15T18:08:41.7545645Z

Does it work the same?

Reply:
Ugh, thanks for spotting this. I started with the update in early January, and they've made another update since then. I had to check all charts again 🤦 The good news is that there aren't any major differences.